opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -18,26 +18,33 @@
 /**
  * Tracer is a simple, thin interface for Span creation and propagation across arbitrary transports.
  */
-public interface Tracer {
+public interface Tracer extends ActiveSpanSource {
```

Comment:
> it's always fine to start a new Span via SpanBuilder.startActive() regardless of whether there's an activeSpan() – not sure if I understand your meaning.

That helper method could be for situations where you don't want to necessary start new span, start it only if there is no active span. Just to keep span tree minimal. But I don't feel very strong either about providing such API... 

`Tracer.adopt` is handy when you need to set active span in some context where you don't have control over executor service etc. 

Some example: https://github.com/redhat-helloworld-msa/ola/blob/82aeeec525bef2102098cd5602d1f13af07a124f/src/main/java/com/redhat/developers/msa/ola/OlaController.java#L118 
It can be done in a better way but people are still using this constructs.